modules/nw-anp-nodes-peer-concept.adoc

```diff
@@ -0,0 +1,232 @@
+<3> Specifies a strong `Deny` egress policy to the internet by setting `networks` values to - 0.0.0.0/0. Make sure you have an allow policy at a higher priority before setting a `Deny` to 0.0.0.0/0 because this will deny all traffic including to API servers.
+====
+
+Collectively the "Example of ANP and BANP using `networks` peers enforces the following egress policy:
+
+* All pods cannot talk to company's intranet DNS servers.
+
+* All pods can talk to rest of the company's intranet.
+
+* All pods can talk to other pods, nodes, and services.
+
+* All pods cannot talk to the internet. Combining the last ANP `Pass` rule and the strong BANP `Deny` rule a guardrail policy is created that secures traffic in the cluster.
+
+[id="combined-nodes-peer-networks-peer-anp"]
+== Using nodes peer and networks peer together
+
+Cluster administrators can combine `nodes` and `networks` peer in your ANP and BANP policies.
+
+.Example of combined `nodes` and `networks` peer
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  name: egress-peer-1 <1>
+spec:
+  egress: <2>
+  - action: "Allow"
+    name: "allow-egress"
+    to:
+    - nodes: <3>
+        matchExpressions:
+        - key: worker-group
+          operator: In
+          values:
+          - workloads # Egress traffic from nodes with label worker-group: workloads is allowed.
+    - networks:
+      - 104.154.164.170/32
+    - pods:
+        namespaceSelector:
+          matchLabels:
+            apps: external-apps
+        podSelector:
+          matchLabels:
+            app: web # This rule in the policy allows the traffic directed to pods labeled apps: web in projects with apps: external-apps to leave the cluster.
+  - action: "Deny"
+    name: "deny-egress"
+    to:
+    - nodes:
+        matchExpressions:
+        - key: worker-group
+          operator: In
+          values:
+          - infra # Egress traffic from nodes with label worker-group: infra is denied.
+    - networks:
+      - 104.154.164.160/32 # Egress traffic to this IP address from cluster is denied.
+    - pods:
+        namespaceSelector:
+          matchLabels:
+            apps: internal-apps
+        podSelector: {}
+  - action: "Pass"
+    name: "pass-egress"
+    to:
+    - nodes:
+        matchExpressions:
+        - key: node-role.kubernetes.io/worker
+          operator: Exists # All other egress traffic is passed to NetworkPolicy or BANP for evaluation.
+  priority: 30 <3>
+  subject: <4>
+    namespaces:
+      matchLabels:
+        apps: all-apps
+----
+<1> Specifies the name of the policy.
+<2> For `nodes` and `networks` peers, you can only use northbound traffic controls in ANP as `egress`.
+<3> Specifies the priority of the ANP, determining the order in which they should be evaluated. Lower priority rules have higher precedence. ANP accepts values of 0-99 with 0 being the highest priority and 99 being the lowest.
```

Comment:
I see two `3's` (by chance :D I hope peer reviewing is also done because I am not very familiar with the annotation bits in docs)

Reply:
I got all mixed up in my numbering when we updated the policy. I fixed it now.